components/price/price.md

```diff
@@ -22,9 +29,9 @@ Price with discount:
 Sizes:
 
     <div>
-      <Price size="xs" value={50153.30} /><br/><br/>
-      <Price size="s" value={50153.30} /><br/><br/>
-      <Price size="m" underlined value={50153.30} /><br/><br/>
-      <Price size="l" underlined value={50153.30} /><br/><br/>
-      <Price size="xl" underlined value={50153.30} /><br/><br/>
+      <Price showAsterisk size="xs" underlined value={50153.30} /><br/><br/>
+      <Price showAsterisk size="s" underlined value={50153.30} /><br/><br/>
```

Comment:
I would suggest to leave few examples without asterisk or add both cases of sizes (with and without asterisk)